src/connectors/lending_connectors/MorphoConnector.sol

```diff
@@ -1,53 +1,78 @@
 // SPDX-License-Identifier: BUSL-1.1
 pragma solidity ^0.8.28;
 
+import "@openzeppelin/contracts/interfaces/IERC20.sol";
 import "../../interfaces/ILendingConnector.sol";
 import "./interfaces/IMorphoBlue.sol";
-import "src/utils/MulDiv.sol";
+import "../../utils/MulDiv.sol";
+import "forge-std/console.sol";
 
 contract MorphoConnector is ILendingConnector {
     using uMulDiv for uint128;
 
     IMorphoBlue public constant MORPHO = IMorphoBlue(0xBBBBBbbBBb9cC5e90e3b3Af64bdAF62C37EEFFCb);
-    IMorphoBlue.MarketParams public marketParams;
+
+    IERC20 public immutable loanToken;
+    IERC20 public immutable collateralToken;
+    address public immutable oracle;
+    address public immutable irm;
+    uint256 public immutable lltv;
     bytes32 public immutable marketId;
 
     constructor(IMorphoBlue.MarketParams memory _marketParams) {
```

Comment:
By the way in future we need to think how to make it in the way that we initialised it and store in every particular vault - not in connector

(we can put it as parameters into vault constructor or make a factory)